dev-python/pytest-qt/pytest-qt-3.3.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="pytest plugin for PyQt4 or PyQt5 applications"
+HOMEPAGE="https://pypi.org/project/pytest-qt https://github.com/pytest-dev/pytest-qt"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc"
+
+RDEPEND="
+	>=dev-python/pytest-3.0.0[${PYTHON_USEDEP}]
+	dev-python/PyQt5[gui,testlib,${PYTHON_USEDEP}]
+	doc? (
+		dev-python/sphinx[${PYTHON_USEDEP}]
+		dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
+	)"
```

Comment:
You should be able to replace this with `distutils_enable_sphinx _build/html sphinx_rtd_theme`.

Reply:
Oops, I had changed this and pushed it to https://github.com/gentoo/gentoo/pull/14007 but I seem to have forgotten to push it to here as well.